packages/govuk-frontend/src/govuk/components/details/_index.scss

```diff
@@ -8,66 +8,7 @@
   }
 
   .govuk-details__summary {
-    // Make the focus outline shrink-wrap the text content of the summary
-    display: inline-block;
-
-    // Absolutely position the marker against this element
-    position: relative;
-
     margin-bottom: govuk-spacing(1);
```

Comment:
Hmm, I can't reproduce this via Browserstack in Edge 15 - 18, how are you testing?
![Details component in Edge 18, showing element styles](https://github.com/alphagov/govuk-frontend/assets/22524634/cc65d1bc-1973-464e-a593-e5a41d679e00)

![Details component in Edge 18, showing computed styles](https://github.com/alphagov/govuk-frontend/assets/22524634/5a368bb1-82e1-43c8-bbc4-4cba3b7c75d0)

Reply:
The style appears in the inspector's style and computed tabs, but if you hover over the element so that the box model is highlighted on the page you can see that the `margin-bottom` value hasn't actually been used, even though it's not crossed out. 

The difference can also be seen if you do a side-by-side comparison with the existing review app—the border of the `<details>` contents has moved up by 5 pixels.

![image](https://github.com/alphagov/govuk-frontend/assets/1253214/bd930b89-24ac-49d2-809a-92d810c66b8a)

I'm not sure why EdgeHTML devtools says the style is being used when it actually isn't, but I'm gonna chalk that up to EdgeHTML devtools maybe not being amazing. 😝

Reply:
Aah, nice spot! My basic trust in devtools is forever tainted.